system/objects/permissions.rst

```diff
@@ -34,7 +34,8 @@ Whenever needed you can restrict access to objects based on the
 About screens
 -------------
 
-Zammad cares about the screen you're going to use the object in.
+Zammad differentiates between several screens, you can use the object attribute
```

Comment:
`, you can use the ...` -> `where object attributes can be used` (also remove the comma).